nRF9160Master/src/WiFi/WiFiHandler.c

```diff
@@ -258,6 +260,29 @@ static void CheckAPConnected(const char *pcResp, bool *pbStatus)
     } 
 }
 
+/**
+ * @brief      : GetAPCredentials
+ * @param [in] : None
+ * @param [out]: None
+ * @return     : WIFI_SSID_PWD
+*/
+char *GetAPCredentials(void)
+{
+    return WIFI_SSID_PWD;
+}
+
+/**
+ * @brief      : SetAPCredentials
+ * @param [in] : pcCredential
+ * @param [out]: None
+ * @return     : None
+*/
+void SetAPCredentials(char *pcCredential)
+{
+    memset(WIFI_SSID_PWD, 0, sizeof(WIFI_SSID_PWD));
```

Comment:
Replace the name of Credential buffer as mentioned above.
Also have a null check here also